### PR TITLE
Add MI to News

### PR DESCRIPTION
**Pull request by martian523**
Replace with new MI class

### CODE REVIEW

File: src/content/news/2023-04-15-mi-course.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: 處境化的基督教倫理
-date: 2023-04-15
-thumbnail: /images/news/2023基督教倫理-445x250.jpg
```
Comment (saintfish) [inline src/content/news/2023-04-15-mi-course.md:4]: Please delete the unused image.

File: src/content/news/2023-04-15-mi-course.md
Comment (saintfish) [inline src/content/news/2023-04-15-mi-course.md:1]: Please change the date in the filename.

**Review by saintfish — CHANGES_REQUESTED** [review]

File: src/content/news/2023-04-15-mi-course.md
Comment (martian523) [reply src/content/news/2023-04-15-mi-course.md:1]: Date changed.

File: src/content/news/2023-04-15-mi-course.md (hunk 1 above)
Comment (martian523) [reply src/content/news/2023-04-15-mi-course.md:4]: I think I did?

Comment (saintfish) [reply src/content/news/2023-04-15-mi-course.md:4]: Did you delete public/images/news/2023基督教倫理-445x250.jpg? Please check if you add the deletion to the commit

Comment (martian523) [reply src/content/news/2023-04-15-mi-course.md:4]: ok. just deleted it. so i have to delete all unused images from now on.

Comment (saintfish) [reply src/content/news/2023-04-15-mi-course.md:4]: If you keep the habit to delete images in the change you delete the reference to them, just like this change, you will never have to clean up unused images.

**Review by saintfish — APPROVED** [review]

**Pull request merged**